### PR TITLE
Ensure expected GOPATH setup in the init script

### PR DESCRIPTION
**Pull request by cmluciano**
Signed-off-by: Christopher M. Luciano <cmluciano@us.ibm.com>

### CODE REVIEW

**Comment by ldemailly** [comment]
that'll help indeed, thx

super nit: "Signed-off-by: Christopher M. Luciano..." maybe you want to update your PR template as it looks a bit odd

**Comment by kyessenov** [comment]
That's GPG signature I think.

**Comment by cmluciano** [comment]
Ya I originally just put this in the commit. I think if I only have 1 commit, this becomes the body of the PR.

**Comment by ldemailly** [comment]
doesn't look like gpg to me :-) anyway, if you can use the template with ```release-note ... section that'd be great - or just say more details about the fix in that spot

File: bin/init.sh [hunk 1]
```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 set -ex
 
+# Ensure expected GOPATH setup
+PDIR=`pwd`
+if [ $PDIR != "$GOPATH/src/istio.io/pilot" ]; then
```
Comment (ldemailly) [inline bin/init.sh]: I should have caught this but this is incorrect. GOPATH doesn't need to be set (defaults to ~/go https://golang.org/cmd/go/#hdr-GOPATH_environment_variable )

Comment (ldemailly) [reply bin/init.sh]: fixing in https://github.com/istio/pilot/pull/1181

**Pull request merged**